static-analysis-plugin/plugin/src/test/groovy/com/novoda/test/PenaltyExtensionSubject.groovy

```diff
@@ -23,20 +22,25 @@ class PenaltyExtensionSubject extends Subject<PenaltyExtensionSubject, PenaltyEx
         }
     }
 
-    private PenaltyExtensionSubject(FailureStrategy failureStrategy, @Nullable PenaltyExtension actual) {
-        super(failureStrategy, actual)
-    }
-
     public static PenaltyExtensionSubject assertThat(PenaltyExtension extension) {
         assertAbout(FACTORY).that(extension)
     }
 
+
+    public static PenaltyExtensionSubject assumeThat(PenaltyExtension extension) {
```

Comment:
No they aren't used, it was just to showcase how this can evolve. I was showing this to @eduardb and then we decided to keep it (despite YAGNI & co).